sw/airborne/firmwares/rotorcraft/main_fbw.c

```diff
@@ -175,9 +178,17 @@ STATIC_INLINE void main_periodic(void)
     }
   }
 
+
+  static uint16_t dv = 0;
+  // TODO make module out of led blink?
   /* set failsafe commands     */
   if (fbw_mode == FBW_MODE_FAILSAFE) {
     SetCommands(commands_failsafe);
+    if (!(dv++ % (PERIODIC_FREQUENCY / 20))) { LED_TOGGLE(3);}
```

Comment:
Pixhawk has this pushbutton with integrated led, that can be mounted somewhere on the airframe. So, I've added `PUSH_BUTTON_LED    ?= 3` to the px4io_2.4.makefile, but that does not seem to do it:

"/home/houjebek/paparazzi/sw/airborne/arch/stm32/led_hw.h:43:35: error: 'LED_PUSH_BUTTON_LED_GPIO_PIN' undeclared (first use in this function)"

How to do this?